### PR TITLE
remove old scala version from travis

### PR DESCRIPTION
**Pull request by britneywright**
additional tweak for #569.

Changes in this pull request:
- remove scala version 2.12.7 from travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: scala
 
 scala:
-  - 2.12.7
   - 2.12.8
```
Comment (rebstar6) [inline .travis.yml]: hmm, should this be 18 actually?

Comment (britneywright) [reply .travis.yml]: 🤦🏽‍♀️

Comment (britneywright) [reply .travis.yml]: oh no, this is right. the other PR was named incorrectly, it's 2.12.8

Comment (rebstar6) [reply .travis.yml]: ah ok cool :)

**Pull request merged**